.circleci/src/@common.yml

```diff
@@ -1 +1,10 @@
 version: 2.1
+
+parameters:
+  run_commit:
+    default: true
+    type: boolean
+
+  run_dev_delivery:
+    default: false
+    type: boolean
```

Comment:
How do you set these params currently? And what is the plan in the future (with potential production deliveries and triggered deliveries)?

Reply:
Parameters can be set via api. 
Plan for the future is still a bit open. Will keep it with manual delivers for now (to meet the deadline) and add automated deliveries after the deadline.